web-animations-2/Overview.bs

```diff
@@ -3366,6 +3553,167 @@ Update the type for the AnimationPlaybackEventInit members.
 
 </div>
 
+<h3 id="the-animationtrigger-interface">The <code>AnimationTrigger</code> interface</h3>
+
+<pre class='idl'>
+[Exposed=Window]
+interface AnimationTrigger {
+  constructor(optional AnimationTriggerOptions options = {});
+  attribute AnimationTimeline timeline;
+  attribute AnimationTriggerType type;
+  attribute TimelineRangeOffset rangeStart;
+  attribute TimelineRangeOffset rangeEnd;
+  attribute TimelineRangeOffset exitRangeStart;
+  attribute TimelineRangeOffset exitRangeEnd;
```

Comment:
I suspect these want to be the same complex type as rangeStart and rangeEnd: https://drafts.csswg.org/web-animations-2/#the-animation-interface

Reply:
done